record_indexer/settings.py

```diff
@@ -13,5 +17,18 @@ def get_auth():
     return AWSV4SignerAuth(
         credentials, os.environ.get("AWS_REGION", "us-west-2"))
 
+
+def print_opensearch_error(r: requests.Response, url: str):
```

Comment:
Not a blocker, but this feel weird in a `settings.py`

Reply:
Agreed, I wasn't sure where it did belong. I suppose I could create a `record_indexer.utils`